src/server/game/Entities/AreaTrigger/AreaTrigger.cpp

```diff
@@ -77,18 +88,55 @@ bool AreaTrigger::CreateAreaTrigger(ObjectGuid::LowType guidlow, uint32 triggerE
 
     SetEntry(triggerEntry);
     SetDuration(spell->GetDuration());
+
+    SetUInt32Value(AREATRIGGER_DURATION, spell->GetDuration());
+
     SetObjectScale(1);
 
     SetGuidValue(AREATRIGGER_CASTER, caster->GetGUID());
     SetUInt32Value(AREATRIGGER_SPELLID, spell->Id);
     SetUInt32Value(AREATRIGGER_SPELLVISUALID, spellVisual);
     SetUInt32Value(AREATRIGGER_DURATION, spell->GetDuration());
+    SetUInt32Value(AREATRIGGER_TIME_TO_TARGET_SCALE, spell->GetDuration()); // Todo
```

Comment:
It has no impact on current implementation (because you send final scale before the areatrigger is visible). You would need to modify scale after it is added to world to notice this